build.gradle

```diff
@@ -61,6 +61,11 @@ dependencies {
     detektPlugins libs.detekt.formatting
 }
 
+wrapper {
+    version '8.7'
+    distributionType Wrapper.DistributionType.BIN
+}
```

Comment:
note: there is no need to add the version. We use the wrapper task only to update the Gradle version so why would we need a particular version in the build script?

Reply:
Why not ?

Reply:
I have explained above why not) What is the point of having a specific version in the wrapper task if this task is only used to update the gradle wrapper to a newer version?